src/script/src/python/coprocessor.rs

```diff
@@ -372,7 +371,11 @@ pub(crate) fn init_interpreter() -> Arc<Interpreter> {
                 let native_module_allow_list = HashSet::from([
                     "array", "cmath", "gc", "hashlib", "_json", "_random", "math",
                 ]);
-                let interpreter = Arc::new(vm::Interpreter::with_init(Default::default(), |vm| {
+                // TODO(discord9): edge cases, can't use "..Default::default" because Settings is `#[non_exhaustive]`
+                // so more in here: https://internals.rust-lang.org/t/allow-constructing-non-exhaustive-structs-using-default-default/13868
+                let mut settings = vm::Settings::default();
+                settings.no_sig_int = true;
```

Comment:
Why set `no_sig_int` to be true?

Reply:
This is to fix the bug of not able to ctrl_c to exit in console #232 which is because RustPython take the SIGINT handler therefore override tokio's `ctrl_c` method hence cause #232 's bug

> Why set `no_sig_int` to be true?